.github/workflows/ci.yml

```diff
@@ -320,6 +320,7 @@ jobs:
       - name: Start HTTP Server (MacOS, no Docker)
         if: runner.os == 'macos'
         run: |
+          mkdir -p /usr/local/etc/nginx
```

Comment:
i'm not sure we should create a directory that nginx doesn't know about, but .  We may have to just download nginx from somewhere...  anyway, i think it might not be worth our time at this stage - until they install nginx on -13 and -14 the same way they have it on -12 (latest).

Reply:
We could do more when[ `macos-14` is stable, now is `beta`](https://docs.github.com/en/actions/using-jobs/choosing-the-runner-for-a-job#choosing-github-hosted-runners).  There are many issues bigger than this.